scripts/help-utils.js

```diff
@@ -139,11 +126,9 @@ const serializeFolderSave = async (json, locale) => {
  * @return {Promise}        [description]
  */
 exports.localizeFolder = async (folder, locale) => {
-    getTranslation(TX_PROJECT, folder.slug, locale, {mode: 'default'})
+    txPull(TX_PROJECT, folder.slug, locale, {mode: 'default'})
```

Comment:
👍 appreciate putting the `JSON.parse` into txPull, so that it's not duplicated/forgotten when getting data.